lib/configure.js

```diff
@@ -54,6 +55,11 @@ module.exports = async () => {
     gitIgnorePatterns.push(tslintConfigFileName);
   }
 
+  // Generate ESLint configuration
+  const eslintConfigFilename = '.eslintrc';
+  await writeFileToCWD(eslintConfigFilename, eslintConfig);
```

Comment:
Confirming, can `.eslintrc` handle the comments in json? By default the comment banner is prepended so just making sure.

Reply:
Yep, it can.